server/routes/index.test.ts

```diff
@@ -46,6 +48,33 @@ describe('GET /', () => {
         expect($('[data-test="view-timetable"] .card__link').text()).toBe('View visits timetable')
         expect($('[data-test="view-timetable"] .card__link').attr('href')).toBe('/timetable')
 
+        expect($('[data-test="block-dates"] .card__link').text()).toBe('Block visit dates')
+        expect($('[data-test="block-dates"] .card__link').attr('href')).toBe('/date-management')
+
+        expect($('[data-test="change-establishment"]').text()).toContain('Change establishment')
+      })
+  })
+
+  it('should not render the block-dates card if feature is disabled', () => {
+    config.features.sessionManagement = false
+    return request(app)
+      .get('/')
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        const $ = cheerio.load(res.text)
+
+        expect($('.card').length).toBe(4)
+
+        expect($('[data-test="book-or-change-visit"] .card__link').text()).toBe('Book or change a visit')
+
+        expect($('[data-test="need-review"] .card__link').text()).toContain('Need review')
+
+        expect($('[data-test="view-visits-by-date"] .card__link').text()).toBe('View visits by date')
+
+        expect($('[data-test="view-timetable"] .card__link').text()).toBe('View visits timetable')
+
+        expect($('[data-test="block-dates"] .card__link').text()).toBe('')
```

Comment:
This is fine but they these are already tested in previous test. So for specifically testing that this card **does not** show then the check of length `4` and an explicit check that the new card is **not** there would be enough: e.g.
```
expect($('[data-test="block-dates"] .card__link').length).toBe(0)
```